lib/tty/command.rb

```diff
@@ -203,7 +204,7 @@ def use_printer(class_or_name, options)
     #
     # @api private
     def find_printer_class(name)
-      const_name = name.to_s.capitalize.to_sym
+      const_name = name.to_s.split('_').map(&:capitalize).join('').to_sym
```

Comment:
For one word printers, the split will simply return an array with one element (the initial string), so it works the same.